packages/mwp-app-render/src/components/uxcapture/UXCapture.jsx

```diff
@@ -11,44 +11,56 @@ type Props = {|
 |};
 
 export const onMark = (mark: string) => {
-	if (!window.newrelic) {
+	if (!(window.newrelic && window.performance)) {
 		return;
 	}
 
-	const now = new Date().getTime();
+	const markStartTime = window.performance
+		.getEntriesByType('mark')
+		.find(entry => entry.name === mark).startTime;
 
 	// Set a marker in the trace details
 	window.newrelic.addToTrace({
 		name: mark,
-		start: now,
+		start: markStartTime,
```

Comment:
Sadaf, are you sure this is the right time for NewRelic?

It is relative to navigationStart and not a full timestamp.

Reply:
Not sure what you mean by `It is relative to navigationStart and not a full timestamp.`

Reply:
UserTiming mark's `startTime` only keeps a number of seconds (e.g. `103.79999998258427`) since navigationStart, which is a full timestamp, equal to number of seconds since epoch (e.g. `1539110917528`)